### PR TITLE
feat: add cypress viewport matrix

### PR DESCRIPTION
**Pull request by SivagamiYagesh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,7 @@
     "typescript": "^4.8.4"
   },
   "scripts": {
-    "cypress": "cypress open",
```
Comment (tlolkema) [inline package.json:13]: We should not have changed anything in the package.json

File: cypress.env.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "VIEWPORT": "MOBILE"
```
Comment (tlolkema) [inline cypress.env.json:2]: Please put this file in .gitignore

**Review by tlolkema — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (SivagamiYagesh) [reply package.json]: Thank you for noticing this change. I wasn't aware of this update.

**Review by aarsenovski — APPROVED** [review]

**Review by tlolkema — APPROVED** [review]

File: cypress/support/commands.ts [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import { any } from 'cypress/types/bluebird'
```
Comment (tlolkema) [inline cypress/support/commands.ts:1]: 🤮

**Review by tlolkema — CHANGES_REQUESTED** [review]

**Review by tlolkema — APPROVED** [review]

**Pull request merged**